examples/default.yaml

```diff
@@ -215,6 +215,12 @@ video:
   # on performance on macOS hosts: https://gitlab.com/qemu-project/qemu/-/issues/334
   # 🟢 Builtin default: "none"
   display: null
+  # VNC (Virtual Network Computing) is a platform-independent graphical
+  # desktop-sharing system that uses the Remote Frame Buffer protocol (RFB)
+  vnc:
+    # VNC display, e.g., "127.0.0.1:0", "host:d", "unix:path", "none"
```

Comment:
This should explain that if you don't know what these options mean, then you want `127.0.0.1:0`.

Reply:
The only issue would be if you started two instances, then one of them would be `127.0.0.1:1`

Reply:
Implemented the "127.0.0.1:0,to=9" now, to pick the first available display